iml-system-test-utils/src/docker.rs

```diff
@@ -0,0 +1,223 @@
+use crate::{iml, CheckedStatus, iml::IML_DOCKER_PATH};
+use iml_systemd;
+use iml_wire_types::Branding;
+use std::{io, thread, time, str};
+use tokio::{
+    fs::{canonicalize, File},
+    io::AsyncWriteExt,
+    process::Command,
+};
+
+pub struct DockerSetup {
+  pub use_stratagem: bool,
+  pub branding: Branding,
+}
+
+pub async fn docker() -> Result<Command, io::Error> {
+    let mut x = Command::new("docker");
+
+    let path = canonicalize(IML_DOCKER_PATH).await?;
+
+    x.current_dir(path);
+
+    Ok(x)
+}
+
+pub async fn deploy_iml_stack() -> Result<(), io::Error> {
+    iml_systemd::start_unit_and_wait("iml-docker.service".into(), 400).await?;
```

Comment:
400 seconds?

Reply:
I experimented with this a bit and 400 is probably excessive but sometimes it can take a while for everything to load up and all of the docker services to be ready.